src/main/java/edu/ucsb/cs156/happiercows/models/CreateCommonsParams.java

```diff
@@ -1,25 +1,31 @@
 package edu.ucsb.cs156.happiercows.models;
 
-import lombok.Data;
+import java.util.Collection;
+
+import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
-import lombok.NoArgsConstructor;
 import lombok.Builder;
-import lombok.AccessLevel;
-
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import org.springframework.security.core.GrantedAuthority;
 
 import edu.ucsb.cs156.happiercows.entities.User;
 
-import java.util.Collection;
-
 @Data
 @AllArgsConstructor
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 @Builder
-public class CreateCommonsParams {
+public class CreateCommonsParams
+{
   private String name;
-  private double cowPrice;
-  private double milkPrice;
-  private double startingBalance;
+  private String cowPrice;
+  private String milkPrice;
+  private String startingBalance;
+  private String year;
+  private String month;
+  private String day;
+  private String hour;
+  private String minute;
+  private String second;
```

Comment:
Maybe using `LocalDateTime` to replace all of these parameters would be cleaner? That way, you wouldn't need to write any parsing logic and it would be easier to use on the frontend (since dates from datepickers are generally always in ISO 8601 format)

Reply:
I can replace them with `LocalDateTime.parse`. I didn't know date pickers give ISO-8601 strings.

https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/LocalDateTime.html#parse(java.lang.CharSequence)